packages/dendron-next-server/pages/vscode/note-preview.tsx

```diff
@@ -34,22 +39,27 @@ function AntLayout(props: React.PropsWithChildren<any>) {
   );
 }
 
-function Note({ engine, ide }: DendronProps) {
+function Note({ engine, ide, initNoteId }: NotePreviewProps) {
   logger.info({
     state: "enter",
   });
+  const [isFirstRender, setFirstRender] = useState(true);
 
   const dispatch = engineHooks.useEngineAppDispatch();
 
   const { noteActive } = ide;
-  const { id: noteId = "9eae08fb-5e3f-4a7e-a989-3f206825d490", contentHash } =
+  let { id: noteId = "9eae08fb-5e3f-4a7e-a989-3f206825d490", contentHash } =
```

Comment:
What's the significance of this hash/note?

Reply:
it was used when testing. when opening this up in chrome, it would show this note by default. let me test this without opening vscode